simulation/traffic_simulator/src/entity/entity_manager.cpp

```diff
@@ -38,12 +39,30 @@ namespace entity
 void EntityManager::broadcastEntityTransform()
 {
   std::vector<std::string> names = getEntityNames();
-  for (const auto & name : names) {
-    geometry_msgs::msg::PoseStamped pose;
-    pose.pose = getMapPose(name);
-    pose.header.stamp = clock_ptr_->now();
-    pose.header.frame_id = name;
-    broadcastTransform(pose);
+  if (isEgoSpawned()) {
+    const auto ego_name = getEgoName();
+    if (entityExists(ego_name)) {
+      broadcastTransform(
+        geometry_msgs::build<geometry_msgs::msg::PoseStamped>()
+          .header(std_msgs::build<std_msgs::msg::Header>().stamp(clock_ptr_->now()).frame_id("ego"))
```

Comment:
```suggestion
          .header(std_msgs::build<std_msgs::msg::Header>().stamp(clock_ptr_->now()).frame_id(ego_name))
```

Reply:
This is the intended implementation.
It is easier to create rviz config if the name “ego” is fixed, so the frame_id “ego” is issued regardless of the name of the ego entity.

Reply:
I see, I understand.

Reply:
I checked well-documented in https://github.com/tier4/scenario_simulator_v2/pull/1238/commits/9b5444900e2ac529131c5bf1ab5d4481465baf49